### DISCUSSION TITLE
Replace all instances of braces in require_once()

### DISCUSSION THREAD

**Discussion by anytizer** [opencart/opencart#11306] [Ideas]
require_once() does not need braces. We can remove them through out the application.

    require_once(DIR_SYSTEM . 'startup.php');

can be replaced with:

     require_once DIR_SYSTEM . 'startup.php';

**Comment by mhcwebdesign** [comment]
Why? What difference does it make?

**Reply by anytizer** [reply]
Simply because these brackets are needless.

Performance achievement will be negligible by removing the bracketed evaluation. It is a matter of style only. A statement from https://pear.php.net/manual/en/standards.including.php suggests not to surround.

I am afraid to say; but it would be a step ahead for minimal code as it grows with PHP 8.
